docs/index.md

```diff
@@ -84,6 +84,11 @@ Also known as the human to bot (H2B) stream API.
 
 ## Changelog
 
+### v1.5.0
+
+* Support for the `n_best_list_length` parameter to get alternatives transcriptions and interpretations
+  when available.
+
```

Comment:
```suggestion
* Drop support for Python <= 3.9

```

Reply:
@rtxm I think you missed my suggestion here

Reply:
argh, the button didn't work.

Reply:
will do another one

Reply:
done directly on main